### PR TITLE
Add support of Tuya Moes temperature and humidity sensor with screen.

### PR DESCRIPTION
**Pull request by PlusPlus-ua**
Support of Tuya Moes temperature and humidity sensor with screen is added. Files ts0201_neo.py and ts0201_zemismart.py are merged into one ts0201.py and new classes added there.

Fix: #1667

### CODE REVIEW

**Comment by PlusPlus-ua** [comment]
This PR resolves issue #1667

**Comment by javicalle** [comment]
Please, rebase your `Tuya_ts0201` branch to allow the merge.

File: zhaquirks/tuya/ts0201.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+class ValueAlarm(t.enum8):
+    """Temperature and humidity alarm values."""
+
+    ALARM_OFF = 0x02
+    MAX_ALARM_ON = 0x01
+    MIN_ALARM_ON = 0x00
+
+
+class TuyaTemperatureHumidityAlarmCluster(CustomCluster):
+    """Neo temperature and humidity alarm cluster (0xE002)."""
```
Comment (javicalle) [inline zhaquirks/tuya/ts0201.py:35]: Also the comment can be changed to """Tuya temperature and ...

**Review by javicalle — APPROVED** [review]

**Comment by javicalle** [comment]
It would be great to add some tests for these alarm attributes and values, but I really don't even know where to start from so I would say that is OK to me.

**Comment by javicalle** [comment]
Edit your first comment and add inside:
```
Fix: #1667
```
This way the issue will be closed one the PR is merged.

File: zhaquirks/tuya/ts0201.py (hunk 1 above)
Comment (PlusPlus-ua) [reply zhaquirks/tuya/ts0201.py]: Agree, have done it.

**Comment by Sirse** [comment]
This device drains battery with ZHA (12% in week). With TuYa hub about 10% in month.

**Comment by Sirse** [comment]
> > This device drains battery with ZHA (12% in week). With TuYa hub about 10% in month.
> 
> So far, mine has drained 5% in 25 days, with ZHA

Mine device is KCTW1Z-01 (Moes round).

**Comment by PlusPlus-ua** [comment]
> Mine device is KCTW1Z-01 (Moes round).

Mine one is also Moes but square. I almost sure that difference is only in LCD screen and plastic case.

**Comment by PlusPlus-ua** [comment]
Mine one today peeled off and falling from 2 m height. As a result, sensor is working, but LCD screen is not :( Be aware and use better two side scotch.

**Comment by FS1961** [comment]
I have the same issue but am not smart enough to do a "quirk". Will this be in an update to HomeAssistant or the ZHA integration soon?

Thanks for your work!

**Comment by Sirse** [comment]
> I have the same issue but am not smart enough to do a "quirk". Will this be in an update to HomeAssistant or the ZHA integration soon?
> 
> 
> 
> Thanks for your work!

In next major HA update i think. U can just download this quirk and drop to your custom_quirk directory (don't forget to enable it in configuration.yaml)

**Pull request merged**